### DISCUSSION TITLE
Feature request: attach multiple site filters to a person

### DISCUSSION THREAD

**Discussion by felinerescue** [sheltermanager/asm3#1574] [Ideas]
Multiple roles can be selected for a user account, but not more than one site. This would be helpful for users who have multiple roles but that we still don't want to see every location site. 
Alternatively, adding the ability to attach location filters onto the user roles.

**Comment by Tetzor** [comment]
Hi, Sites are really only intended to easily separate users to the single site where they work for organisations that have more than one site/branch. Location filters are better suited for for choosing the visible locations on a per user basis. I can see how attaching location filters to roles could be useful and have added a ticket for this.  Jon

**Comment by felinerescue** [comment]
Thank you!
________________________________
